helm/hello-world/values.yaml

```diff
@@ -18,13 +18,16 @@ fullnameOverride: ""
 serviceAccount:
   # Specifies whether a service account should be created
   create: true
+  # Automatically mount a ServiceAccount's API credentials?
+  automount: true
```

Comment:
The pod doesn't really need a ServiceAccount token, right?

```suggestion
  automount: false
```

Reply:
Well, it had before and I did not want to change the default behavior.

Reply:
But yeah, disabled it by default now. :)